common/appointment/get.ts

```diff
@@ -112,7 +109,6 @@ const getAppointmentsForUserFromNow = async (userId: User['userID'], take: numbe
     const appointmentsFromNow = await prisma.lecture.findMany({
         where: {
             isCanceled: false,
-            subcourse: { published: true },
```

Comment:
I think what you actually want is OR: [{ subcourse: null }, { subcourse: { ... }]